cmake/onnxruntime_providers.cmake

```diff
@@ -6,11 +6,13 @@ file(GLOB_RECURSE onnxruntime_providers_srcs CONFIGURE_DEPENDS
   "${ONNXRUNTIME_ROOT}/core/providers/cpu/*.cc"
 )
 
-file(GLOB_RECURSE onnxruntime_contrib_ops_srcs CONFIGURE_DEPENDS
```

Comment:
Can we keep this? It was added by someone outside our team and he seemed to have good reasons to use this.

Reply:
done